setup.py

```diff
@@ -47,7 +47,7 @@
         "aiohttp",
         "nest_asyncio",
         "pyLD",
-        "pyshacl",
+        "pyshacl==0.11.6.post1",
```

Comment:
Can we use [v0.11.6](https://github.com/RDFLib/pySHACL/releases/tag/v0.11.6) instead of [v0.11.6.post1](https://github.com/RDFLib/pySHACL/releases/tag/v0.11.6.post1) ?

@annakristinkaufmann Did you try with v0.11.6 too as part of #44

Reply:
@MFSY Yes, v0.11.6 worked as well as far as I remember.

Reply:
Why using an older version?

Reply:
v0.11.6 looks newer (even though both releasing were made the same day) from the tag listing: https://github.com/RDFLib/pySHACL/releases

Reply:
I think v0.11.6.post1 is newer: https://github.com/RDFLib/pySHACL/compare/v0.11.6...v0.11.6.post1

Reply:
Okay. The tag listing is misleading then.